python/hawkey/tests/tests/test_reldep.py

```diff
@@ -61,6 +61,9 @@ def test_custom_querying(self):
         reldep = hawkey.Reldep(self.sack, "P-lib = 3-3")
         q = hawkey.Query(self.sack).filter(provides=reldep)
         self.assertLength(q, 1)
+        reldep = hawkey.Reldep(self.sack, "P-lib == 3-3")
```

Comment:
Could you just add a comment here also stating the `==` was a mistake and will be eventually removed? Otherwise LGTM.

Reply:
Added.